packages/manager/src/components/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,95 @@
+import { Typography, useTheme } from '@mui/material';
+import { default as _Avatar } from '@mui/material/Avatar';
+import * as React from 'react';
+
+import AkamaiWave from 'src/assets/logo/akamai-wave.svg';
+import { usePreferences } from 'src/queries/profile/preferences';
+import { useProfile } from 'src/queries/profile/profile';
+
+import type { SxProps } from '@mui/material';
+
+export const DEFAULT_AVATAR_SIZE = 28;
+
+export interface AvatarProps {
+  /**
+   * Optional background color to override the color set in user preferences
+   * */
+  color?: string;
+  /**
+   * Optional height
+   * @default 28px
+   * */
+  height?: number;
+  /**
+   * Optional styles
+   * */
+  sx?: SxProps;
+  /**
+   * Optional username to override the profile username; will display the first letter
+   * */
+  username?: string;
+  /**
+   * Optional width
+   * @default 28px
+   * */
+  width?: number;
+}
+
+/**
+ * The Avatar component displays the first letter of a username on a solid background color.
+ * For system avatars associated with Akamai-generated events, an Akamai logo is displayed in place of a letter.
+ */
+export const Avatar = (props: AvatarProps) => {
+  const {
+    color,
+    height = DEFAULT_AVATAR_SIZE,
+    sx,
+    username,
+    width = DEFAULT_AVATAR_SIZE,
+  } = props;
+
+  const theme = useTheme();
+
+  const { data: preferences } = usePreferences();
+  const { data: profile } = useProfile();
+
+  const _username = username ?? profile?.username ?? '';
+  const isAkamai = /^Linode$|^lke-service-account*/.test(_username);
```

Comment:
Could this work to avoid the regex?

```suggestion
  const isAkamai = _username === 'Linode' || _username.startsWith('lke-service-account');
```

Reply:
Sure, I didn't think this regex was too bad, but I'm fine with avoiding it altogether. Changed in [bb98f35](https://github.com/linode/manager/pull/10859/commits/bb98f35612e882f943283749496aeb72782a4cee).

While making the change, I also caught a bug fixed in [10f0742](https://github.com/linode/manager/pull/10859/commits/10f07423d801dc2bc0a4e2cb2f7b1834a15ae3a0) related to the event `username` being `null`:
| Bug | Fix |
| --- | --- |
|![Screenshot 2024-09-06 at 10 14 02 AM](https://github.com/user-attachments/assets/a98bc6ea-84c3-4eba-98cd-a74abf98df44) | ![Screenshot 2024-09-06 at 10 13 44 AM](https://github.com/user-attachments/assets/b2e5e4a5-4c80-4821-8919-36d61b2bd659)|